blocks/init/src/blocks/components/menu/menu-style.scss

```diff
@@ -0,0 +1,2 @@
+@import './styles/menu-horizontal';
```

Comment:
be consistant with the folder structure.

it should be:
- menu:
-- components
---horizontal
----menu-horizontal-style.scss
---vertical
----menu-vertical-style.scss

If someone wants to completely change the structure of menu in lets say horizontal than it can put the menu-horizontal.php file inside the horizontal folder and her is done. He doesn't need to change the folder structure.